src/D2L.Bmx/ParameterDescriptions.cs

```diff
@@ -3,7 +3,7 @@ namespace D2L.Bmx;
 internal static class ParameterDescriptions {
 	public const string Org = "Okta org short name or domain name";
 	public const string User = "Okta username";
-	public const string Password = "Okta password";
+	public const string Password = "Okta password for username '{0}' in org '{1}'";
```

Comment:
`ParameterDescriptions.Password` should remain unchanged. It's meant to be succinct description of the "password" parameter itself.
It's also used in the `--help` message where it really doesn't make sense to show it like this (including org & username).

Reply:
Ah okay I didn't think about it showing up in `--help`

Reply:
should check all usages (find references) before such a change